src/sentry/integrations/pipeline.py

```diff
@@ -126,6 +126,7 @@ def finish_pipeline(self):
                 extra={
                     "error_message": str(e),
                     "error_status": getattr(e, "code", None),
+                    "organization_id": self.organization.id,
```

Comment:
`self.organization` has type `RpcOrganization | None`

```suggestion
                    "organization_id": self.organization.id if self.organization else None,
```